docs/transaction-buffer.md

```diff
@@ -0,0 +1,16 @@
+# Transaction Buffer
+
+## Introduction
+Failed send transaction requests would be buffered in redis server.
```

Comment:
I can not tell that whether the requests are sent or unsent or both.
The description needs to be rewritten.

Reply:
what kinds of description you what me to append?

Reply:
Send is a verb. I think it can not be written like that.
It might be `Failed transaction sending requests`.